CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 #### Features & Updates
 
-* None
+* New Example: CleanseSwiftUIExample.
```

Comment:
2 whitespaces needed at the end to create a newline.

Reply:
okay